components/buttons/button.css

```diff
@@ -0,0 +1,111 @@
+.btn{
+    
+    padding: 0.5rem 1rem;
+    margin: 1.25rem 0;
+    /* border-radius: 0.3rem; */
+    background-color: transparent;
+    /* border: none; */
+    border: 2px solid grey;
+    font-size: 1rem;
+    text-align: center;
+    width: 12rem;
+    /* outline: none; */
+    cursor: pointer;
+}
+.btn-solid{
+    background-color: var(--btn-color);
+}
+.btn-outline{
+    border: 2px solid #0dafa0;
+    color: #159673;
+}
+.btn-bottom{
+    border: none;
+    text-decoration: underline;
+}
+/* 
+DISBALE BTN
+*/
+.btn-disable{
+    pointer-events: none;
+    opacity: 0.5;
+}
+/* 
+CSS FOR BTN SIZE
+*/
+.lg-btn{
+    font-size: 1.1rem;
+    padding: 0.7rem 1.5rem;
+    min-width: 80px;
+}
+.sm-btn{
+    font-size: 0.9rem;
+    padding: 0.3rem 0.7rem;
+    width: 112px;
+}
+/* 
+CSS FOR TOGGLE BTN
+*/
+.btn-toggle{
+    width: 26px;
+    height: 44px;
+    position: relative;
+    cursor: pointer;
+    display: block;
+}
+/* .btn-toggle:focus-within{
+    box-shadow: 0 0 0 2px black;
+} */
+.checkbox{
+    height: 0;
+    width: 0;
+    margin: 0;
+    opacity: 0;
+}
+.btn-toggle .btn-slider{
+    position: absolute;
+    top:0;
+    left:9px;
+    height: 26px;
+    width: 26px;
+    background-color: var(--btn-color);
+    border-radius: 50%;
+    transition: 0.4s;
+}
+.btn-toggle .btn-slider::before{
+    position: absolute;
+    content: "";
+    top:0;
+    left: 9px;
+    width: 26px;
+    height: 26px;
+    border-radius: 50%;
+    background-color: rgb(226, 226, 226);
+    transition: 0.4s;
+}
+.btn-toggle .checkbox:checked + .btn-slider::before{
```

Comment:
Try to avaoid multiple nested classes. Single nesting is fine.

Reply:
sure i will try to avoid this mistakes